test/numa/README.md

```diff
@@ -0,0 +1,39 @@
+# CRI Resource Manager - NUMA node tests
+
+## Usage
+
+```
+[VAR=VALUE...] ./run.sh MODE
+```
+
+Get help with `./run.sh`.
+
+## Two modes: `test` and `play`
+
+`test` mode runs fast and, by default, cleans up everything after test
+run: the virtual machine with the contents will be lost.
+
+`play` mode runs slower and, by default, leaves the virtual machine
```

Comment:
This can be achieved by altering either execution (typing) speed or clean selection:
`speed=1000 ./run.sh demo`
or
`cleanup=0 ./run.sh test`

I'll document these to README...

Reply:
README updated to give a hint there're options for cleanup and speed.